src/main/java/asynchronous/typing/Daily.java

```diff
@@ -0,0 +1,52 @@
+package asynchronous.typing;
+
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.JDA;
```

Comment:
JDA isn't used in this class, so you can omit this line and lines 18 and 26 as well.